plasmapy/atomic/tests/test_parsing.py

```diff
@@ -9,7 +9,7 @@
     _case_insensitive_aliases,
     _case_sensitive_aliases,
     _parse_and_check_atomic_input,
-    _call_string,
+    _particle_call_string,  # duplicate with utils.pytest_helpers.error_messages.call_string?
```

Comment:
What's that about?

Reply:
I made a more general `plasampy.utils.pytest_helpers.call_string` function which creates a string that replicates the call from a supplied callable, args, and kwargs to help with error messages, so this private function for a specific case is no longer necessary.  I might as well go back and finish this change that I started but did not finish.